packages/@aws-cdk/aws-s3-deployment/test/integ.bucket-deployment.ts

```diff
@@ -27,6 +28,17 @@ class TestBucketDeployment extends cdk.Stack {
       destinationKeyPrefix: 'deploy/here/',
       retainOnDelete: false, // default is true, which will block the integration test cleanup
     });
+
+    const bucket3 = new s3.Bucket(this, 'Destination3');
```

Comment:
I am expecting to see a change in `.expected.json`

Reply:
Please see previous comment. I'm unable to run integration tests. Someone else will have to jump in and update the `.expected.json`.